frontend/views/containers/payments/RecordPayment.vue

```diff
@@ -0,0 +1,232 @@
+<template lang='pug'>
+modal-base-template(ref='modal' :fullscreen='true' class='has-background')
+  .c-payment-form(v-if='!donePayment')
+    .c-header
+      i18n.is-title-2.c-title(tag='h2') Record payments
+
+    .c-content
+      form.card.c-card(
+        @submit.prevent='submit'
+        novalidate='true'
+      )
+        i18n.has-text-bold.c-title(tag='h3') Who did you send money to?
+        payments-list(
+          :titles='tableTitles'
+          :payments='fakeStore.paymentsDistribution'
+          paymentsType='edit'
+        )
+
+        .c-footer
+          .c-footer-info
+            i18n.has-text-bold(
+              tag='h4'
+              :args='{ optional: `<span class="has-text-small has-text-1 has-text-normal">(${L("optional")})</span>`}'
+            ) Add a note {optional}
+
+            i18n.has-text-small.has-text-1(
+              tag='p'
+            ) Leave a message to the group members selected above.
+          .c-footer-action
+            input.switch#showComment(
+              type='checkbox'
+              name='displayComment'
+              @change='displayComment = !displayComment'
+            )
+            label.sr-only(for='displayComment') Toggle comment box
+
+        transition(name='slidedown')
+          label.field(v-if='displayComment')
+            i18n.sr-only.label Leave a message
+            textarea.c-comment(
+              rows='4'
+            )
+
+        .buttons
+          i18n.is-outlined(
+            tag='button'
+            @click='closeModal'
+          ) Cancel
+
+          i18n.is-success(
+            tag='button'
+            @click='submit'
+            :disabled='$v.form.$invalid'
+            :args='{number: "0"}'
+          ) Register {number} payment
+
+  .c-payment-success(v-else)
+    svg-success
+    i18n.is-title-2.c-title(tag='h2') Your payments were recorded
+    banner-simple(severity='info')
+      i18n Please consider supporting the development of Group Income by sending a donation!
+
+    i18n.is-outlined(
+      tag='button'
+      @click='closeModal'
+    ) Close
+</template>
+
+<script>
+import ModalBaseTemplate from '@components/modal/ModalBaseTemplate.vue'
+import PaymentsList from '@containers/payments/PaymentsList.vue'
+import L from '@view-utils/translations.js'
+import { validationMixin } from 'vuelidate'
+import SvgSuccess from '@svgs/success.svg'
+import BannerSimple from '@components/banners/BannerSimple.vue'
+
+export default {
+  name: 'RecordPayment',
+  mixins: [validationMixin],
+  data () {
+    return {
+      displayComment: false,
+      form: {},
+      donePayment: false,
+      // Temp until modal params are merged
+      fakeStore: {
+        paymentsDistribution: [{
+          to: 'pierre',
+          amount: 910.99,
+          amountFormatted: '$910.99'
```

Comment:
Although it's a fakeStore, we should still remind ourselves to use the `displayWithCurrency` function by using it here too